website/pages/docs/commands/scaling/policy-list.mdx

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+page_title: 'Commands: scaling policy list'
+sidebar_title: policy list
+description: |
+  Display all Nomad scaling policies.
+---
+
+# Command: scaling policy list
+
+List is used to detail all scaling policies stored in Nomad.
```

Comment:
```suggestion
List is used to list all scaling policies stored in Nomad.
```

Reply:
this returns a list-stub representation, which is why i suggested the changes around list language